src/components/Missions/Missions.js

```diff
@@ -0,0 +1,27 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getMissions, joinMission } from '../../redux/Missions/missionsSlice';
+import MissionList from './MissionList';
+
+const Missions = () => {
+  const dispatch = useDispatch();
+  const missions = useSelector((state) => state.missions);
+  useEffect(() => {
+    if (!missions.length) {
+      dispatch(getMissions());
+    }
+  });
```

Comment:
- [OPTIONAL] I think it would be better to add `missions` and `dispatch` to the dependency array of useEffect because they are used inside the useEffect Hook 👍

```
  useEffect(() => {
    if (!missions.length) {
      dispatch(getMissions());
    }
  }, [missions, dispatch]);
```